ml-agents/mlagents/trainers/ppo/trainer.py

```diff
@@ -254,7 +254,7 @@ def create_torch_policy(
             behavior_spec,
             self.trainer_settings,
             condition_sigma_on_obs=False,  # Faster training for PPO
-            separate_critic=behavior_spec.action_spec.is_continuous(),
+            separate_critic=behavior_spec.action_spec.continuous_size > 0,
```

Comment:
Can this parameter be removed, and just make the decision based on the `behavior_spec` in `TorchPolicy.__init__`?

Reply:
I think this is necessary because SAC always uses a separate critic where as PPO depends on the continuous space

Reply:
BTW, this was changed in `master` - all of the networks use a separate critic as of Release 10.

Reply:
I'll cut it when I merge master into the staging branch